system-parachains/collectives/collectives-polkadot/src/fellowship/origins.rs

```diff
@@ -19,6 +19,7 @@
 use super::ranks;
 pub use pallet_origins::*;
 
+#[allow(unreachable_patterns)]
```

Comment:
Would be nice to have an issue to remove this when we have upgraded to the polkadot-sdk version that fixes this.